javascript/ql/lib/semmle/javascript/frameworks/Spife.qll

```diff
@@ -0,0 +1,412 @@
+/**
+ * Provides classes for working with [Spife](https://github.com/npm/spife) applications.
+ */
+
+import javascript
+import semmle.javascript.frameworks.HTTP
+
+/**
+ * Provides classes for working with [Spife](https://github.com/npm/spife) applications.
+ */
+module Spife {
+  /**
+   * A call to a Spife method that sets up a route.
+   */
+  private class RouteSetup extends API::CallNode, Http::Servers::StandardRouteSetup {
```

Comment:
This doesn't work, as there is no API node for the relevant call.  

In general there is not API nodes for everything, only the things you can reach using a chain of calls to the `API-graph` predicates.   

```suggestion
  private class RouteSetup extends DataFlow::CallNode, Http::Servers::StandardRouteSetup {
```

Some things will stop compiling, and you should add some inline tests to the `javascript/ql/test/library-tests/frameworks/Spife/lib/routes/index.js` file.  

To be clear, the use of `API::Node` in `ReplyCall` is fine.